staging/src/k8s.io/kubectl/pkg/cmd/delete/delete_flags.go

```diff
@@ -155,6 +159,9 @@ func (f *DeleteFlags) AddFlags(cmd *cobra.Command) {
 	if f.Raw != nil {
 		cmd.Flags().StringVar(f.Raw, "raw", *f.Raw, "Raw URI to DELETE to the server.  Uses the transport specified by the kubeconfig file.")
 	}
+	if f.Confirm != nil {
+		cmd.Flags().BoolVar(f.Confirm, "confirm", *f.Confirm, "If true, delete resource only when user confirms")
```

Comment:
`--interactive` `-i` to mimic what other UNIX tools are already using, see this:
```
$ rm -i /tmp/gopls.18112-*
rm: remove regular file '/tmp/gopls.18112-1GiB-nonames.zip'? n
rm: remove regular file '/tmp/gopls.18112-1GiB-withnames.zip'? y
rm: remove regular file '/tmp/gopls.18112-2GiB-nonames.zip'? y
rm: remove regular file '/tmp/gopls.18112-2GiB-withnames.zip'? n
```

On top of that, I was initially thinking about just one by one, but there are probably pros and cons for that vs what @mpuckett159 proposed elsehwere, where he said:
>  Yes to all (A)
>  Yes to this one (y)
>  No to this one (n)
>  No to all (C? As in cancel. I dunno this one I'm not so sure about)

With a few changes:
1. No need for cancel, ctrl+c is enough of a cancel for CLI, no need to create yet another one ;-) 
2. I'm debating between `A` and `y`, for the latter we print one by one, for the former the user won't have an option to know in advance (unless he'll run `--dry-run` first) what `A` will actually remove. But I think I can live with that. 

We could probably discuss that last element on our next sig-cli call, maybe? 

Lastly, should we hide this behind a env var until we gain confidence it's good enough, or at least we should mark the flag as alpha, to give a clean signal to users that we might change it in the next release.

Reply:
Doh yeah ctrl+c makes way more sense haha.